codesign/certificates.go

```diff
@@ -29,13 +29,13 @@ var (
 		"Mac Developer",                       // type: "Mac Development"
 		"3rd Party Mac Developer Application", // type: "Mac App Distribution"
 		"Developer ID Application",            // type: "Developer ID Application"
+		"Apple Development",                   // type: "iOS Development for Xcode 11", id: "83Q87W3TGH"
```

Comment:
What is the for?

Reply:
This is the id of the cert type on the dev portal, but as we do not use them, i will remove that part of the comment